include/util/FitsComparator.h

```diff
@@ -0,0 +1,36 @@
+//
+// Created by Samuel Cappon on 3/5/20.
+//
+
+#ifndef FITSCOMPARATOR_H
+#define FITSCOMPARATOR_H
+
+#include <fitsio.h>
+#include <vector>
+#include <string>
+#include <map>
+using namespace std;
```

Comment:
Preferably, please do not use `using namespace std` because it will create namespace pollution. https://softwareengineering.stackexchange.com/questions/236404/what-is-using-namespace-pollution
This will give you some hints.